include/CL/sycl/item.hpp

```diff
@@ -112,6 +115,12 @@ class item : boost::equality_comparable<item<Dimensions>> {
   */
   void set(id<Dimensions> Index) { global_index = Index; }
 
+  /** Returns an item with same dimensions but offset set to 0 */
+  operator item<Dimensions, true> () const {
+    static_assert(with_offset == false, "get_offset only callable with has_offset = true");
+    item<Dimensions, true> new_item { this->get_range(), this->get_id(), 0 };
+    return new_item;
```

Comment:
I do not know if you did it for clarification purpose.

Otherwise you can probably just write
```
return { this->get_range(), this->get_id(), 0 };
```